app/services/database_health_checker.rb

```diff
@@ -0,0 +1,26 @@
+module DatabaseHealthChecker
+  module_function
+
+  Summary = Struct.new(:healthy, :result) do
+    def as_json(*args)
+      to_h.as_json(*args)
+    end
+  end
+
+  # @return [Summary]
+  def check
+    Summary.new(true, simple_query)
+  rescue => err
+    Summary.new(false, err.message)
```

Comment:
Any concerns about making the unfiltered error message public? Does this make it possible for something like the connection URL to show up in healthcheck response body?